### ISSUE TITLE
Launch and close Nexus Mods App

### ISSUE THREAD

**Issue by Greg-Nexus** [Nexus-Mods/NexusMods.App#444]
# User story

**As a** user

**I want to** to launch the app from an installation on my system

**So that** I can start using the app

## Requirements
Once a user has downloaded and installed the app, they will need to be able to launch and open it.

The scope of this issue is basic:

1. User finds the app executable
2. User opens app executable
3. App launches to 'Home' screen
4. User closes the app
5. App closes

Out of scope (to be done later, in separate issues):

- App icons (needs design)
- Launch/loading splash screen

## Design
N/A

## Related Issues

- **Blocked by:** https://github.com/Nexus-Mods/NexusMods.App/issues/376
